vue/components/ui/molecules/select-checkboxes/select-checkboxes.vue

```diff
@@ -0,0 +1,95 @@
+<template>
+    <select-ripe
+        class="select-checkboxes"
+        v-bind:options="selectOptions"
+        v-bind:value="'checkbox-group'"
+        v-bind="{
+            autoScroll: false,

```

Comment:
Is this really needed?

Reply:
Yes if we don't want to have it scroll to the bottom of the checkboxes-group